yb-voyager/cmd/exportDataDebezium.go

```diff
@@ -59,23 +59,24 @@ func prepareDebeziumConfig(partitionsToRootTableMap map[string]string, tableList
 	default:
 		return nil, nil, fmt.Errorf("invalid export type %s", exportType)
 	}
-	fmt.Printf("num tables to export: %d\n", len(tableList))
-	utils.PrintAndLog("table list for data export: %v", tableList)
 	tableNameToApproxRowCountMap := getTableNameToApproxRowCountMap(tableList)
 
 	var dbzmTableList, dbzmColumnList []string
 	for _, table := range tableList {
+		t := table.Qualified.MinQuoted
+		if source.DBType == POSTGRESQL && table.SchemaName.MinQuoted == "public" {
+			t = table.ObjectName.MinQuoted
+		}
+		if leafPartitions[t] != nil {
+			//In case of debezium offline migration of PG, tablelist should not have root and leaf both so not adding root table in table list
+			continue
+		}
 		dbzmTableList = append(dbzmTableList, table.Qualified.Unquoted)
```

Comment:
Wait you don't seem to be using t. Is that intended? t  is just for skipping root tables?

Reply:
Yes